aisummary/pyproject.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "indico-plugin-AIsummary"
```

I'd use all lowercase for the name:
```suggestion
name = "indico-plugin-ai-summary"
```